flow/scripts/set_place_density.tcl

```diff
@@ -7,6 +7,7 @@ if {[info exist ::env(PLACE_DENSITY_LB_ADDON)]} {
   if {$place_density > 1.0} {
     utl::error FLW 24 "Place density exceeds 1.0 (current PLACE_DENSITY_LB_ADDON = $::env(PLACE_DENSITY_LB_ADDON)). Please check if the value of PLACE_DENSITY_LB_ADDON is between 0 and 0.99."
   }
+  puts "Placement density is $place_density, computed from PLACE_DENSITY_LB_ADDON $::env(PLACE_DENSITY_LB_ADDON) and lower bound $place_density_lb"
```

Comment:
It would be cleaner if this was a proc in util that returned a value. In macro placement the place density is logged, so no need to print it for macro placement